report/report.go

```diff
@@ -297,6 +302,10 @@ func MakeReport() Report {
 			WithTag(Camera).
 			WithLabel("volume snapshot data", "volume snapshot data"),
 
+		Job: MakeTopology().
+			WithShape(StorageSheet).
```

Comment:
Just a reminder to replace this with `DottedTriangle` once we get the new shape from `ui-components`.

Reply:
Yes!